actin/operational/upload_report_for_kaiko

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
what do you think about calling this `share_report_with_nki? I think the verb share is used elsewhere in this context and kaiko is more like the middle man.